### PR TITLE
chore: Use `import` syntax

### PR DESCRIPTION
**Pull request by DominicKramer**

### CODE REVIEW

File: src/configuration.ts [hunk 1]
```diff
@@ -15,8 +15,8 @@
  */
 
 var env = process.env;
-var has = require('lodash.has');
-var is = require('is');
+import * as has from 'lodash.has';
```
Comment (jinwoo) [inline src/configuration.ts]: Move these imports to the top?

File: src/google-apis/auth-client.ts [hunk 2]
```diff
@@ -26,7 +26,7 @@ export interface Common {
 
 const common: Common = require('@google-cloud/common');
 const pkg = require('../../../package.json');
-var is = require('is');
+import * as is from 'is';
```
Comment (jinwoo) [inline src/google-apis/auth-client.ts]: Ditto

File: src/logger.ts [hunk 3]
```diff
@@ -15,8 +15,8 @@
  */
 /*jshint bitwise: false*/
```
Comment (jinwoo) [inline src/logger.ts]: Not related to this PR but can this be line deleted?

File: src/populate-error-message.ts [hunk 4]
```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
-var has = require('lodash.has');
-var is = require('is');
+import * as has from 'lodash.has';
+import * as is from 'is';
```
Comment (jinwoo) [inline src/populate-error-message.ts]: It'd be good if imports are sorted by the package names. I think `gts fix` does that, doesn't it?

File: src/request-extractors/hapi.ts [hunk 5]
```diff
@@ -15,8 +15,8 @@
  */
 // jscs:disable requireEarlyReturn
 
-var has = require('lodash.has');
-var is = require('is');
+import * as has from 'lodash.has';
+import * as is from 'is';
```
Comment (jinwoo) [inline src/request-extractors/hapi.ts]: Ditto

File: src/request-extractors/manual.ts [hunk 6]
```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
-var has = require('lodash.has');
-var is = require('is');
+import * as has from 'lodash.has';
+import * as is from 'is';
```
Comment (jinwoo) [inline src/request-extractors/manual.ts]: Ditto

**Comment by DominicKramer** [comment]
@jinwoo Right now `gts check` is disabled.  My goal with these PRs in the middle of the translation process is to make minimal style changes and only have syntax changes.  The last PRs in translating the `src` directory will re-enable `gts check` and run `gts fix`.  At that time, your suggestions should be addressed automatically.

**Pull request merged**